src/assets/wise5/authoringTool/project-authoring-lesson/project-authoring-lesson.component.html

```diff
@@ -12,8 +12,17 @@
     class="projectItemTitleDiv"
     fxLayoutAlign="start center"
     fxLayoutGap="10px"
-    (click)="setCurrentNode(item.id)"
+    (click)="setCurrentNode(lesson.id)"
   >
-    <node-icon-and-title [nodeId]="item.id" [showPosition]="showPosition"></node-icon-and-title>
+    <node-icon-and-title [nodeId]="lesson.id" [showPosition]="showPosition"></node-icon-and-title>
   </div>
 </div>
+<div *ngFor="let childId of lesson.ids">
+  <project-authoring-step
+    [step]="idToNode[childId]"
+    (selectNodeEvent)="selectNodeEvent.emit($event)"
+    [nodeTypeSelected]="nodeTypeSelected"
```

Comment:
ProjectAuthoringLesson and ProjectAuthoringStep shouldn't have to know about NodeTypeSelected- they should only need to know if they need to disable selection?
```suggestion
    [disableSelection]="nodeTypeSelected === 'group'"
```